www/js/app.js

```diff
@@ -1453,29 +1451,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                 }
             }
         }
-
-        function loadJavaScriptJQuery() {
-            $('#articleContent').contents().find('script[data-kiwixurl]').each(function() {
-                var script = $(this);
-                var scriptUrl = script.attr("data-kiwixurl");
-                // TODO check that the type of the script is text/javascript or application/javascript
-                var title = uiUtil.removeUrlParameters(decodeURIComponent(scriptUrl));
-                selectedArchive.getDirEntryByTitle(title).then(function(dirEntry) {
-                    if (dirEntry === null) {
-                        console.log("Error: js file not found: " + title);
-                    } else {
-                        selectedArchive.readBinaryFile(dirEntry, function (fileDirEntry, content) {
-                            // TODO : JavaScript support not yet functional [kiwix-js #152]
-                            uiUtil.feedNodeWithBlob(script, 'src', content, 'text/javascript');
-                        });
-                    }
-                }).catch(function (e) {
-                    console.error("could not find DirEntry for javascript : " + title, e);
-                });
-            });
-        }
-
```

Comment:
@ram690514 Although we don't currently use the above function, it is a placeholder. It could be commented out, though it is harmless to leave it in uncommented. If it improves our codefactor score, then comment it out, but please leave the code for future devs to look at.

Reply:
okay , i found the function currently unused , that's why i removed the function .. but  I  had replaced it and made it as a comment

Reply:
OK